docs/data/material/getting-started/templates/checkout/getCheckoutTheme.tsx

```diff
@@ -0,0 +1,816 @@
+      main: green[400],
+      dark: green[800],
+      ...(mode === 'dark' && {
+        light: green[400],
+        main: green[500],
+        dark: green[700],
+      }),
+    },
+    grey: {
+      50: gray[50],
+      100: gray[100],
+      200: gray[200],
+      300: gray[300],
+      400: gray[400],
+      500: gray[500],
+      600: gray[600],
+      700: gray[700],
+      800: gray[800],
+      900: gray[900],
+    },
+    divider: mode === 'dark' ? alpha(gray[600], 0.3) : alpha(gray[300], 0.5),
+    background: {
+      default: '#fff',
+      paper: gray[50],
+      ...(mode === 'dark' && { default: gray[900], paper: gray[800] }),
+    },
+    text: {
+      primary: gray[800],
+      secondary: gray[600],
+      ...(mode === 'dark' && { primary: '#fff', secondary: gray[400] }),
+    },
+    action: {
+      selected: `${alpha(brand[200], 0.2)}`,
+      ...(mode === 'dark' && {
+        selected: alpha(brand[800], 0.2),
+      }),
+    },
+  },
+  typography: {
+    fontFamily: ['"Inter", "sans-serif"'].join(','),
+    h1: {
+      fontSize: 60,
+      fontWeight: 600,
+      lineHeight: 78 / 70,
+      letterSpacing: -0.2,
+    },
+    h2: {
+      fontSize: 48,
+      fontWeight: 600,
+      lineHeight: 1.2,
+    },
+    h3: {
+      fontSize: 42,
+      lineHeight: 1.2,
+    },
+    h4: {
+      fontSize: 36,
+      fontWeight: 500,
+      lineHeight: 1.5,
+    },
+    h5: {
+      fontSize: 20,
+      fontWeight: 600,
+    },
+    h6: {
+      fontSize: 18,
+    },
+    subtitle1: {
+      fontSize: 18,
+    },
+    subtitle2: {
+      fontSize: 16,
+    },
+    body1: {
+      fontWeight: 400,
+      fontSize: 15,
+    },
+    body2: {
+      fontWeight: 400,
+      fontSize: 14,
+    },
+    caption: {
+      fontWeight: 400,
+      fontSize: 12,
+    },
+  },
+});
+
+export default function getCheckoutTheme(mode: PaletteMode): ThemeOptions {
+  return {
+    ...getDesignTokens(mode),
+    components: {
+      MuiAccordion: {
+        defaultProps: {
+          elevation: 0,
+          disableGutters: true,
+        },
+        styleOverrides: {
+          root: ({ theme }) => ({
+            padding: 8,
+            overflow: 'clip',
+            backgroundColor: '#fff',
+            border: '1px solid',
+            borderColor: gray[100],
+            ':before': {
+              backgroundColor: 'transparent',
+            },
+            '&:first-of-type': {
+              borderTopLeftRadius: 10,
+              borderTopRightRadius: 10,
+            },
+            '&:last-of-type': {
+              borderBottomLeftRadius: 10,
+              borderBottomRightRadius: 10,
+            },
+            ...(theme.palette.mode === 'dark' && {
+              backgroundColor: gray[900],
+              borderColor: gray[800],
+            }),
+          }),
+        },
+      },
+      MuiAccordionSummary: {
+        styleOverrides: {
+          root: ({ theme }) => ({
+            border: 'none',
+            borderRadius: 8,
```

Comment:
There are some border-radius definitions throughout this theme; maybe we should define a base token for it and use it throughout for consistency?

Reply:
In the default theme, we only have `shape.borderRadius`, and this makes it hard to use tokens like `sm`, `md`, `lg` on the template without breaking things when `Material Design 2` is selected. For this approach, it'd be necessary to have an `if` every time we use border radius. 

Unless you are only considering the theme file, and the border-radius in it—I believe that would be possible, as long as we don't use anything like `shape.borderRadius.sm` anywhere outside `getCheckoutTheme.tsx`.